spec/controllers/application_controller_spec.rb

```diff
@@ -51,6 +53,7 @@
         it "has set connection" do expect(Base).to have_received(:connection) end
         it "has not called session create" do expect(Session).to_not have_received(:create) end
         it "has set user_id" do expect(controller.request.session[:auth_user][:user_id]).to eq user_id end
+        it "has set provider_count" do expect(controller.request.session[:auth_user][:provider_count]).to eq nil end
```

Comment:
These blocks should use braces (`{}`). How did the lines above get in ...

Reply:
🙈 🙉 🙊

Reply:
You let him put those in!!

Reply:
Actually, I take it back about using `{}`s ... these should be multi-line blocks.